client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/index.tsx

```diff
@@ -40,6 +39,13 @@ jest.mock( 'calypso/state/sites/hooks/use-site-global-styles-status', () => ( {
 	} ),
 } ) );
 
+jest.mock( '@automattic/data-stores', () => ( {
```

Comment:
I'd love to not have custom React queries in `@automattic/data-stores`, it just feels wrong. Not for this PR of course!

Reply:
I do agree! – What do you think of a (new) package named `@automattic/queries` or, if we want to have a package which contains "data" related functions rename `@automattic/data-stores` to just `@automattic/data`? I tend to like the first one more. Curious to hear your thoughts :)

Reply:
I like `@automattic/queries` 👍  I guess the ones that we _need_ to use externally can be added there. Otherwise, it could be more straightforward for them to live under `client/data`. As we have discussed before, having all queries in 1-2 places can definitely help with migrations and keeping better control over code quality and ownership.